src/adapters/mc/ssh/LinuxSshServerSecurityBaseline.mof

```diff
@@ -305,6 +324,7 @@ instance of OsConfigResource as $OsConfigResource18ref
 instance of OsConfigResource as $OsConfigResource19ref
 {
     ResourceID = "Ensure that the SSH port is configured";
+    RuleId = "1310042D-193F-414A-A7E3-B2C4E89AA537";
```

Comment:
Just curious. Is this a new `RuleId`? It looks different from others (also uppercase).

Reply:
The new rule ids are all in clean all uppercase characters. There are total of 6 such rules here. This is one subtle way to tell which ones are, for example.

Reply:
Please reactivate if you still have questions or comments